SECURITY.md

```diff
@@ -4,16 +4,22 @@ We take security very seriously at GoDaddy. We appreciate your efforts to
 responsibly disclose your findings, and will make every effort to acknowledge
 your contributions.
 
-## Where should I report security issues?
+## Disclosure policy
```

Comment:
I would highly advice keeping this file insync with the version provided the our OSWG.

Reply:
Not feeling strong about it, but I what about specific security information like https://github.com/godaddy/warehouse.ai/blame/master/SECURITY.md#L23. Enforcing all files to be similar just for the sake of it feels restricting.

Reply:
This is now in sync